backend/benefit/users/tests/test_gdpr_api.py

```diff
@@ -1,41 +1,45 @@
 import json
+from typing import Optional
 
 import pytest
 from django.contrib.auth import get_user_model
 from django.test import override_settings
 from django.urls import reverse
 
-from shared.common.tests.factories import UserFactory
+from shared.common.tests.factories import HelsinkiProfileUserFactory
 from users.models import format_date
 
 User = get_user_model()
 
 
-def extract_val(resp: dict, key: str) -> str:
+def extract_value(gdpr_api_response: dict, key: str) -> Optional[str]:
```

Comment:
As much as I would like to use type hints myself, their use is not very consistent at the moment, so perhaps we should make decision to either use them everywhere or nowhere.

Reply:
I would like to see them more, also :+1: I think they should be used as much as possible. [Helsinki city's coding standards](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/OS/pages/7737901085/Koodausstandardit+coding+standards) also suggests that they should be used.

There are cases of course where they add maybe too little value for the effort. For example, some test class test methods. Also, some libraries like `factories` have poor typing support, especially with Django models.

I would say that all the new and touched code should primarily have them, but leave option for not using them, depending on the case.

Btw. this specific function is out of date and is removed in current head.